resources/admin/classes/domain/Resource.php

```diff
@@ -815,12 +815,12 @@ public function searchQuery($whereAdd, $orderBy = '', $limit = '', $count = fals
 									LEFT JOIN AcquisitionType AT ON RA.acquisitionTypeID = AT.acquisitionTypeID
 									LEFT JOIN Status S ON R.statusID = S.statusID
 									LEFT JOIN User CU ON R.createLoginID = CU.loginID
-									LEFT JOIN ResourcePurchaseSiteLink RPSL ON R.resourceID = RPSL.resourceID
-									LEFT JOIN ResourceAuthorizedSiteLink RAUSL ON R.resourceID = RAUSL.resourceID
-									LEFT JOIN ResourceAdministeringSiteLink RADSL ON R.resourceID = RADSL.resourceID
-									LEFT JOIN ResourcePayment RPAY ON R.resourceID = RPAY.resourceID
-									LEFT JOIN ResourceNote RN ON R.resourceID = RN.resourceID
-									LEFT JOIN ResourceStep RS ON R.resourceID = RS.resourceID
+									LEFT JOIN ResourcePurchaseSiteLink RPSL ON R.resourceID = RPSL.resourceAcquisitionID
+									LEFT JOIN ResourceAuthorizedSiteLink RAUSL ON R.resourceID = RAUSL.resourceAcquisitionID
+									LEFT JOIN ResourceAdministeringSiteLink RADSL ON R.resourceID = RADSL.resourceAcquisitionID
+									LEFT JOIN ResourcePayment RPAY ON RA.resourceAcquisitionID = RPAY.resourceAcquisitionID
+									LEFT JOIN ResourceNote RN ON RA.resourceAcquisitionID = RN.entityID
```

Comment:
One problem is that the ResourceNote table "entityID" field only exists in the 3.0.0/002-350.sql upgrade script, so it (and maybe more) needs to be added to the main install.sql (right? is that still our approach to fresh installations?).

Reply:
I'm not sure. I thought we decided that fresh installs would process install.sql *and* the upgrade scripts. Can someone confirm this?

Reply:
@veggiematts, correct. When installing the scripts process `install.sql` and all upgrade sql files as long as the upgrade files are contained within a version directory, e.g. `/$MODULE/install/protected/$VERSION`

Here is the [PR](https://github.com/coral-erm/coral/pull/354)

Reply:
@queryluke @veggiematts Got it. Thanks for the clarification.